README.md

```diff
@@ -6,7 +6,7 @@
 [![Discord](https://img.shields.io/discord/1153072414184452236?logo=discord&style=flat)](https://discord.gg/pAbnFJrkgZ)
 [![Twitter](https://img.shields.io/twitter/url/https/twitter.com/cloudposse.svg?style=social&label=Follow%20%40Chi_Wang_)](https://x.com/Chi_Wang_)
```

Comment:
are we updating the twitter handle too in this PR, if the page is created already. otherwise looks good to me.

Reply:
Will udpate later.